components/wizard/Wizard.js

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { classnames } from '../../helpers/component';
 
 const Wizard = ({ step = 0, steps = [], hideText = false }) => {
     return (
-        <div className={`wizard-container ${hideText ? 'wizard-container--noTextDisplayed' : ''}`}>
+        <div className={classnames(['wizard-container', hideText ? 'wizard-container--noTextDisplayed' : ''])}>
```

Comment:
the homogeneity comment again: `hideText ? 'wizard-container--noTextDisplayed' : ''` --> `hideText && 'wizard-container--noTextDisplayed'`